src/GaugeController.sol

```diff
@@ -18,6 +18,9 @@ contract GaugeController {
     // Events
     event NewType(string mame, int128 type_id);
     event NewGauge(address indexed gauge_address, int128 gauge_type);
+    event NewGaugeWeight(address indexed gauge_address, uint256 time, uint256 weight, uint256 total_weight);
+    event NewTypeWeight(int128 type_id, uint256 time, uint256 weight, uint256 total_weight);
+    event VoteForGauge(uint256 time, address user, address gauge_address, uint256 weight);
```

Comment:
nit: add index on user and gauge_address
Also move `time` after the indexed args to avoid order mismatch

Reply:
order and parameters of events is exactly being followed like the curve contracts